lecture_1/task_1/main.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/sethgrid/pester"
+)
+
+type application struct {
+	Name string
+	Age int
+	Passed  bool
+	Skills []string
+}
+
+const applications = "https://run.mocky.io/v3/f7ceece5-47ee-4955-b974-438982267dc8"
+
+func linearBackoff(retry int) time.Duration {
+	return time.Duration(retry) * time.Second
+}
+
+func containsSkills(Skills []string) bool{
```

Comment:
A minor detail. but try naming variables with lowercase (i.e., `skills` instead of `Skills`).